lightning/src/chain/channelmonitor.rs

```diff
@@ -5591,6 +5591,17 @@ impl<Signer: EcdsaChannelSigner> ChannelMonitorImpl<Signer> {
 				OnchainEvent::FundingSpendConfirmation { commitment_tx_to_counterparty_output, .. } => {
 					self.funding_spend_confirmed = Some(entry.txid);
 					self.confirmed_commitment_tx_counterparty_output = commitment_tx_to_counterparty_output;
+					if self.alternative_funding_confirmed.is_none() && !self.pending_funding.is_empty() {
```

Comment:
When would we get this event when `self.alternative_funding_confirmed` is `Some`? Is there some re-org case that could cause this?